sdk/spring/azure-spring-boot/src/test/resources/test-private-key.txt

```diff
@@ -0,0 +1,30 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Private keys should not be checked in. New key can be generated when the test is run.

Reply:
Will remove it.